src/main/scala/org/allenai/plugins/CoreRepositories.scala

```diff
@@ -28,8 +24,7 @@ object CoreRepositories {
     val defaults = Seq(
       spray,
       Resolver.jcenterRepo,
-      typesafeReleases,
-      ai2PublicReleases
```

Comment:
I'm wary of removing `typesafeReleases`--mostly because I don't know what impact it will have.  I'd hate for everyone in every project to have to remember to add that back in.